packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -102,35 +107,37 @@ You can remove the registration objects as follows.
 
 IdP Discovery enables you to route users to different third-party IdPs that are connected to your Okta org. Users can federate back into the primary org after authenticating at the IdP. This feature still functions, but you no longer need to enable the link for users to initialize the route. Instead, you can configure a routing rule with the application context.
 
-You can remove the [IdP Discovery](https://github.com/okta/okta-signin-widget#idp-discovery) JavaScript object in the widget as follows.
+You can remove the `idpDiscovery` JavaScript object in the widget as follows.
 
 ![Displays the IDP Discovery object in JavaScript to remove](/img/SIW_Upgrade_Config_Change2.png)
 
 ### OpenID Connect/social authentication
 
-When [external Identity Providers (IdPs)](https://github.com/okta/okta-signin-widget#openid-connect) are used with Social Login, the supported IdPs (Google, Facebook, Apple, Microsoft, and LinkedIn) are declared with a type, get distinct styling and get default i18n text, while other entries receive general styling and require you to provide text. Each IdP can have additional CSS classes added through an optional `className` property. This feature still functions, however, it's no longer the preferred method to enable the link for users to initialize the route. You can replace this method by configuring a routing rule with the application context.
+When [external Identity Providers (IdPs)](/docs/guides/identity-providers/) are used for Social Login, the supported IdPs (such as Google, Facebook, Apple, Microsoft, and so on) are declared with a `type` property. These supported IdPs get distinct styling and default i18n text, while other custom entries receive general styling and require you to provide text. Each IdP can have additional CSS classes added through an optional `className` property. This feature still functions, however, it's no longer the preferred method to enable the link for users to initialize the route. You can replace this method by configuring a routing rule with the application context.
```

Comment:
I think you need a semi-colon after "functions" in the last sentence. Or break it up into two.